website/docs/reference/appsmith-framework/context-object.md

```diff
@@ -240,7 +252,86 @@ If you have defined custom scopes in your identity provider, the information ass
 </dd>
 </dd>
 
-## geolocation `object`
+### theme `object`
+
+<dd>
+
+This object contains the details of the theme properties applied to the application. You can use this object to set certain properties in widgets to be aligned with the app theme or to write custom logic.
+
+```js
+{{appsmith.theme}}
+```
+
+The theme object has the following attributes:
+
+#### color
+
+#### primaryColor `string`
+
+<dd>
+
+It refers to the app's primary color set in the theme section of the app settings. To access this value, you can use the following code snippet:
+
+```js
+{{appsmith.theme.colors.primaryColor}}
+```
+</dd>
+
+#### backgroundColor `string`
+
+<dd>
+
+The `backgroundColor` property refers to the background color set in the theme section of the app settings. To access the value of background color, you can use the following code snippet:
+
+
+```js
+{{appsmith.theme.colors.backgroundColor}}
+```
+
+</dd>
+
+#### borderRadius
+
+#### appBorderRadius `string`
+
+<dd>
+
+It refers to the border radius set in the theme section of the app settings. To access its value, you can use the following code snippet:
+
+```js
+{{appsmith.theme.borderRadius.appBorderRadius}}
+```
+</dd>
+
+#### boxShadow
+
+#### appBoxShadow `string`
+
+<dd>
+
+It refers to the box shadow set in the theme section of the app settings. You can access the value of `appBoxShadow` using the following snippet: 
+
+```js
+{{appsmith.theme.boxShadow.appBoxShadow}}
+```
+
+</dd>
+
+#### fontFamily
+
+#### appFont `string`
+
+<dd>
+
+It refers to the font family set for the app in the theme section for app settings. You can access the value of `appFont` using the following snippet: 
+
+```js
+{{appsmith.theme.fontFamily.appFont}}
+```
+</dd>
+</dd>
+
+### geolocation `object`
```

Comment:
@bharath31 as per our discussion, the existing geolocation attributes are not clearly explained.  A gentle reminder to provide a draft for these and the missing attributes. cc @jnikhila @danciaclara